### DISCUSSION TITLE
.net Quartz not firing the jobs after publishing the code

### DISCUSSION THREAD

**Discussion by loganayaki23061994** [quartznet/quartznet#1499] [Q&A]
Hi Team,

We are using .net Quartz for scheduling the jobs in IIS and we are handling 3 jobs in single application.
The issue is whenever we do code change and publish to IIS, job is not triggering automatically. some times, it automatically stops.
we tried the Quartz versions of 3.2.4 and 3.3.3
We have configure the app pool setting like below
1. Start Mode - Always running
2. Idle Timeout -0
3. Regular Time Interval -0

Pls help to run the Quartz job continuously.

**Comment by lahma** [comment]
Please search the discussions and issues, we've gone trough this too many times.

**Reply by Jazayery** [reply]
Hi
I didn't find any solution for this problem, If you have any guide please send the link.
Thanks
